charts/apisix-dashboard/Chart.yaml

```diff
@@ -31,12 +31,12 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.3.0
+version: 0.4.1
```

Comment:
why using `0.4.1` version ?

Reply:
First, we released version `2.10` of the dashboard, but it had some issues, so we went further and released a patch version of `2.10.1`. We skipped the `2.10` binary release, so I think it's a little better to skip the `0.4.0` version code. What do you think?

Reply:
It seems that we don't have a mapping relationship between the version of apisix-dashboard and its helm chart.

Reply:
I think it is more appropriate to release 0.4.0. Because Helm chart does not have a direct strong mapping relationship with APISIX Dashboard.

Reply:
OK, I will change it.